src/repos.json

```diff
@@ -46,6 +47,12 @@
     { "language": "ruby", "repo": "googleapis/gapic-generator-ruby", "isTeamIssue": true},
     { "language": "ruby", "repo": "GoogleCloudPlatform/ruby-docs-samples"},
     { "language": "ruby", "repo": "GoogleCloudPlatform/getting-started-ruby"},
-    { "language": "ruby", "repo": "googleapis/ruby-spanner-activerecord", "isTeamIssue": true}
+    { "language": "ruby", "repo": "googleapis/ruby-spanner-activerecord", "isTeamIssue": true},
+
+    { "language": "java", "repo": "GoogleCloudPlatform/getting-started-java", "isTeamIssue": true},
+    { "language": "python", "repo": "GoogleCloudPlatform/getting-started-python", "isTeamIssue": true},
+    { "language": "go", "repo": "GoogleCloudPlatform/golang-samples", "isTeamIssue": true},
+
+    { "language": "java", "repo": "GoogleCloudPlatform/cloud-bigtable-examples", "isTeamIssue": true}
```

Comment:
Could you reorganize the additions to their respective language group?

Reply:
"GoogleCloudPlatform/cloud-bigtable-examples" is a multi-language repo, so I left it on it's own. (It's getting java applied because there really isn't support for multi-langauges).

I moved the Cloud SQL connector repos in line with the others.